agent/custom/general_navigation_action.py

```
@@ -96,265 +95,169 @@
+    pts = [(px - hw, py - hh), (px + hw, py - hh), (px - hw, py + hh), (px + hw, py + hh)]
+    return all(cv2.pointPolygonTest(VISIBLE_POLY, (float(pxx), float(pyy)), True) >= margin
+               for pxx, pyy in pts)
+
+
+def locate_quest_near(img, tpl_bgr, px, py, radius=LOCAL_RADIUS, min_score=MT_MIN_SCORE):
+    """在 (px,py) 附近局部窗口内用目标素材多尺度 matchTemplate 定位名称条
+    (处理 YOLO 漏检: 预测到位但识别列表无目标)
+    返回 (bx, by, bw, bh, score) 名称条左上角+尺寸+分数; 未命中返回 None"""
+    import cv2
+    H, W = img.shape[:2]
+    x0 = max(0, int(px - radius)); x1 = min(W, int(px + radius))
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)